script/clone_job.pl

```diff
@@ -301,7 +301,7 @@ sub clone_job {
         }
     }
     else {
-        die "failed to create job: ", pp($tx->res->body);
+        die "failed to create job: please make sure your http proxy is running (apache, nginx, etc.) ", pp($tx->res->body);
```

Comment:
ah, I see. So you at least know that the string `pp($tx->res->body)` was empty so you could do

```perl
else {
    die "Failed to create job, empty response. Make sure your HTTP proxy is running, e.g. apache, nginx, etc." unless $tx->res->body;
    die "Failed to create job: ", pp($tx->res->body);
}
```